lib/sequelize.js

```diff
@@ -667,22 +632,16 @@ module.exports = (function() {
    *
    * @see {Model#build} for more information about callee.
    */
-  Sequelize.prototype.query = function(sql, callee, options, replacements) {
+  Sequelize.prototype.query = function(sql, callee, options) {
```

Comment:
Don't you want to deprecate/remove the callee too? It's already possible to use options.instance instead (undocumented though).